scalastyle-config.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not a big fan of scalastyle. It [has rotted](https://github.com/scalastyle/scalastyle/graphs/contributors) and everybody is using only wartremover. Also, it's style rules are very frustrating and just plain wrong in many cases.